test/functions_test.py

```diff
@@ -186,8 +124,16 @@
 ...                return input_func_dict[item1]
 ...            else:
 ...                return input_func_dict["error"]
-...    for index in pyrgg.params.PYRGG_ENGINE_PARAMS:
-...        item1, item2 = pyrgg.params.PYRGG_ENGINE_PARAMS[index]
+...    for index in pyrgg.params.PYRGG_ENINGE_PARAMS:
```

Comment:
`ENINGE` or `ENGINE`?

Reply:
fixed in [4728ff1](https://github.com/sepandhaghighi/pyrgg/pull/160/commits/4728ff19de9d07c9b01b1d9c80eba786bd80e685).